luzer/luzer.c

```diff
@@ -300,6 +300,10 @@ TestOneInput(const uint8_t* data, size_t size) {
 	lua_State *L = get_global_lua_state();
 
 	char *buf = malloc(size + 1 * sizeof(*buf));
+	if (!buf) {
+		perror("malloc");
+		abort();
```

Comment:
I suppose something like `exit()` will be better.

Reply:
Updated:

```diff
--- a/luzer/luzer.c
+++ b/luzer/luzer.c
@@ -310,7 +310,7 @@ TestOneInput(const uint8_t* data, size_t size) {
        char *buf = malloc(size + 1 * sizeof(*buf));
        if (!buf) {
                perror("malloc");
-               abort();
+               exit(EXIT_FAILURE);
        }
        memcpy(buf, data, size);
        buf[size] = '\0';
```